src/app/forms/forms.route.js

```diff
@@ -0,0 +1,130 @@
+(function() {
+  'use strict';
+
+  angular.module('uvpBuilderWeb.forms')
```

Comment:
same thing, when you have more than one method call to an object, move each one to a different line.
``` javascript
angular
  .module()
  .config()
```